docs/concepts/resource-manager.md

```diff
@@ -734,6 +734,110 @@ spec:
 The respective pods don't need any additional labels.
 If the annotation's value is empty (`[]`) then all ports are allowed.
 
+#### Services Exposed via `Ingress` Resources
+
+The controller can optionally be configured to watch `Ingress` resources by specifying the pod and namespace selectors for the `Ingress` controller.
+If this information is provided, it automatically creates `NetworkPolicy` resources allowing the respective ingress/egress traffic for the backends exposed by the `Ingress`es.
+This way, neither custom `NetworkPolicy`s nor custom labels must be provided.
+
+The needed configuration is part of the component configuration:
+
+```yaml
+controllers:
+  networkPolicy:
+    enabled: true
+    concurrentSyncs: 5
+  # namespaceSelectors:
+  # - matchLabels:
+  #     kubernetes.io/metadata.name: default
+    ingressControllerPeer:
+      namespace: default
+      podSelector:
+        matchLabels:
+          foo: bar
+```
+
+As an example, let's assume that above `gardener-resource-manager` `Service` was exposed via the following `Ingress` resource:
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: gardener-resource-manager
+  namespace: a
+spec:
+  rules:
+  - host: grm.foo.example.com
+    http:
+      paths:
+      - backend:
+          service:
+            name: gardener-resource-manager
+            port:
+              number: 443
```

Comment:
In `Ingress`es, the service port is referenced.

Reply:
Thanks for pointing out that the `port` -> `targetPort` conversion happens under the covers.

Maybe this could be added as a remark to the documentation as it might not be obvious if you do not know the `gardener-resource-manager` service details by heart.

Reply:
Done with https://github.com/gardener/gardener/pull/7636/commits/094f2e8ef319ef66c240d7de88f3a65859a197b9